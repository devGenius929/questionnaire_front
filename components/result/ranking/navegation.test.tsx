import React from 'react';
import renderer from 'react-test-renderer';
import Navegation from './navegation';

describe('Navegation', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Navegation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
