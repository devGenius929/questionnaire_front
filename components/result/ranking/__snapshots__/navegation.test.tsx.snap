// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navegation renders correctly 1`] = `
<div
  className="cxsubmenusistema"
>
  <ul
    className="submenusistema"
  >
    <li
      id="ranking"
    >
      <a
        href="/resultado/ranking"
      >
        Ranking
      </a>
    </li>
    <li
      id="exportacao"
    >
      <a
        href="/resultado/exportacao"
      >
        Exportação
      </a>
    </li>
  </ul>
</div>
`;
